### PR TITLE
10: Responsive Design 2

### PR DESCRIPTION
**Pull request by gabrielizalo**
# Summay

* Updated html and scss files to change page layout with Responsive Design

# Trello Card

https://trello.com/c/sGoc8A5H/15-dise%C3%B1o-responsive

# Screenshot

html css training 2018-10-03 18-27-07

html css training 2018-10-03 18-27-22

### CODE REVIEW

File: maquetado/scss/_footer.scss [hunk 1]
```diff
@@ -33,3 +33,15 @@
     text-decoration: underline;
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  .foo-final {
```
Comment (henryzarza) [inline maquetado/scss/_footer.scss:37]: The content isn't center. Center it, please. You look the design

File: maquetado/scss/_tech.scss [hunk 2]
```diff
@@ -40,3 +40,22 @@
   font-size: $text-small;
   text-align: center;
 }
+
+@media only screen and (max-width: 1024px) {
+  .technologies {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Delete this property

File: maquetado/scss/_tech.scss [hunk 3]
```diff
@@ -40,3 +40,22 @@
...
+  }
+
+  .technology {
+    border-bottom: $green 2px solid;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Remember change the order `2px solid $green`

File: maquetado/scss/_titles.scss [hunk 4]
```diff
@@ -18,3 +17,9 @@
     z-index: -1;
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  .title {
+    max-width: 180px;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Change this for `60%`

File: maquetado/scss/_titles.scss [hunk 5]
```diff
@@ -1,7 +1,6 @@
 .title {
   align-items: center;
```
Comment (henryzarza) [inline maquetado/scss/_titles.scss]: Remember center the content

**Review by henryzarza — CHANGES_REQUESTED** [review]
Good job

**Review by henryzarza — APPROVED** [review]
God job!

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**